test/secrets_manager_example_test.go

```diff
@@ -0,0 +1,96 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+	"github.com/stretchr/testify/assert"
+)
+
+// This test suite deploys the secrets-manager module from examples/secrets-manager.
+// The test is broken into "stages" so you can skip stages by setting environment variables (e.g.,
+// skip stage "apply" by setting the environment variable "SKIP_apply=true"), which speeds up iteration when
+// running this test over and over again locally.
+func TestSecretsManagerExample(t *testing.T) {
+	t.Parallel()
+
+	// The folder where we have our Terraform code
+	workingDir := "../examples/secrets-manager"
+
+	// At the end of the test, undeploy the secrets using Terraform
+	defer test_structure.RunTestStage(t, "destroy", func() {
+		terraformOptions := test_structure.LoadTerraformOptions(t, workingDir)
+
+		terraform.Destroy(t, terraformOptions)
+	})
+
+	// Provision the secrets using Terraform
+	test_structure.RunTestStage(t, "apply", func() {
+		awsRegion := aws.GetRandomStableRegion(t, []string{"us-east-1", "eu-west-1"}, nil)
+		test_structure.SaveString(t, workingDir, "awsRegion", awsRegion)
+		deployUsingTerraform(t, awsRegion, workingDir)
+	})
+
+	// Validate that the secrets are configured properly
+	test_structure.RunTestStage(t, "validate", func() {
+		validateSecretsContainSecrets(t, workingDir)
+	})
```

Comment:
Does the function `RunTestStage` look for the environment variable? Otherwise how are these two skipped if an environment variable is passed in?